.github/workflows/build.yaml

```diff
@@ -201,6 +201,46 @@ jobs:
           name: ${{matrix.platform}}-lr-logs
           path: '*-logs.tar.gz'
 
+  libtester-deb-install-test:
+    name: libtester deb install test
+    needs: [d, Build, dev-package]
+    if: always() && needs.dev-package.result == 'success'
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ubuntu20] # Assuming want to do both ubuntu20 and ubuntu22 at some point so setting up with matrix
+    runs-on:  ["self-hosted", "enf-x86-lowtier"] # not sure if this warrants a better machine, but start here
+    container: ${{fromJSON(needs.d.outputs.p)['ubuntu20'].image}}
+    steps:
+      - name: Download cdt
+        uses: AntelopeIO/asset-artifact-download-action@v2
+        with:
+          owner: AntelopeIO
+          repo: cdt
+          file: 'cdt_.*amd64.deb'
+          target: main
+          artifact-name: cdt_ubuntu_package_amd64
+          token: ${{github.token}}
+      - name: Download leap-dev
+        uses: actions/download-artifact@v3
+        with:
+          name: leap-dev-ubuntu20-amd64
+      - name: Install cdt and leap-dev Packages
+        run: |
+          apt install -y ./*.deb
+          apt-get -y install cmake
+          rm ./*.deb
+      - name: checkout reference-contracts
+        uses: actions/checkout@v3
+        with:
+          repository: AntelopeIO/reference-contracts
+          path: src
+      - name: Build & Test reference-contracts
+        run: |
+          cmake -S src -B build -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTS=On
+          cmake --build build -- -j $(nproc)
+          ctest --test-dir build/tests --output-on-failure -j $(nproc)
```

Comment:
It seems like the tests aren't running
https://github.com/AntelopeIO/leap/actions/runs/5222841684/jobs/9429031167?pr=1259#step:7:500
What's going on here is that the cmake version included in ubuntu 20.04 doesn't have the `--test-dir` parameter and for some crazy insane reason ctest doesn't error out on unknown command line arguments.

The reason this works over in reference-contracts is because it just uses github's runner image which has [a whole bunch of stuff already built in](https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2004-Readme.md) including... cmake 3.26

So will need to change this line to `cd build/tests && cmake --output-on-failure -j $(nproc)` or such.

(it also probably means over in reference-contracts [this line](https://github.com/AntelopeIO/reference-contracts/blob/76197b4bc60d8dc91a5d65ecdbf0f785e982e279/.github/workflows/build.yaml#L88) is unnecessary)

Reply:
Thanks.  I was digging into why the tests weren't running and this was a great head start. I appreciate it.

Reply:
Done